server/routes/upload.js

```diff
@@ -0,0 +1,47 @@
+const Profile = require("../models/Profile");
+const express = require('express');
+const AWS = require('aws-sdk');
+const multer = require('multer');
+const   multerS3 = require('multer-s3');
+const router = express.Router()
+require('dotenv').config();
+
+const s3 = new AWS.S3({
+  accessKeyId: process.env.AWS_ID,
+  secretAccessKey: process.env.AWS_KEY
+});
+
+var upload = multer({
```

Comment:
```suggestion
const upload = multer({
```

Let's resort to using ES6 syntax (const/let instead of var)